Topic-1-HTML/Task 2/C1.html

```diff
@@ -4,34 +4,36 @@
   <meta charset="UTF-8">
   <meta name="author" content="Eric Vernaschi">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Task 1 Ex B</title>
+  <title>Task 2 Ex C</title>
 </head>
 <body>
   <header>
     <h1>Sing Up</h1>
   </header>
-  <form>
-    <p>First Name:</p>
-    <input type="text" placeholder="Your name" required>
-    <p>Last Name:</p>
-    <input type="text" placeholder="Your last name" required>
-    <p>Email:</p>
-    <input type="email" placeholder="Your Email" required>
-    <p>Birth Date:</p>
-    <input type="date" required>
-    <p>Choose Your Favorite Sport:</p>
-    <select required>
-      <option value="">None</option>
-      <option value="tennis">Tennis</option>
-      <option value="soccer">Soccer</option>
-      <option value="hockey">Hockey</option>
-    </select>
-    <br>
-    <p>Small Bio</p>
-    <textarea placeholder="Max Length 100 characters." maxlength="100"></textarea>
-    <br>
-    <button type="submit" value="signup">Sign Up</button>
-    <button type="reset" value="reset">Clear All Fields</button>
-  </form>
+  <main role="main">
+    <form>
+      <p>First Name:</p>
+      <input type="text" aria-label="fName" placeholder="Your name" required>
```

Comment:
_Minor:_ remember that screen readers will announce the aria-labels, the content should be descriptive, my advice is to use the same text that you are using inside the `p` elements for describing the input,